types/params_test.go

```diff
@@ -0,0 +1,34 @@
+package types_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/tendermint/tendermint/types"
+)
+
+func TestConsensusParamsValidation(t *testing.T) {
+	c1 := new(types.ConsensusParams)
```

Comment:
why not table tests?

Reply:
This seemed simpler, usually I'd use table tests but here they seemed simpler. However, I can add table tests since they aren't tightly coupled to the previous lines and don't require saving context of what previously happened. Thanks for pointing this out :)